web-marketplace/src/components/organisms/ChooseCreditsForm/ChooseCreditsForm.tsx

```diff
@@ -0,0 +1,209 @@
+                          className="pl-10 hover:opacity-80"
+                          rel="noreferrer"
+                        >
+                          Learn more »
+                        </a>
+                      </p>
+                    }
+                  />
+                ))}
+              </RadioGroup>
+            </div>
+          )}
+
+          {/* advance settings */}
+          <div>
+            <div className="grid grid-rows-1 py-20">
+              <div className="flex flex-col">
+                <TextButton
+                  className="w-[193px] text-xs font-['Muli'] text-green-400 font-extrabold uppercase text-brand-400 self-start bg-transparent border-none p-0 text-left"
+                  textSize="sm"
+                  onClick={toggleAdvancedSettings}
+                >
+                  <span className="text-base w-10 inline-block mr-5">
+                    {advanceSettingsOpen ? '-' : '+'}
+                  </span>
+                  Advanced settings
+                </TextButton>
+                {advanceSettingsOpen && (
+                  <div
+                    className={`${
+                      advanceSettingsOpen ? 'max-h-[300px]' : 'max-h-0'
+                    } flex flex-grow justify-end items-center font-['Lato'] text-base flex-col mt-4 overflow-hidden transition-all duration-500 ease-in-out max-h-0`}
+                  >
+                    <p className="self-start text-sm ">
+                      Choose specific credit vintages{' '}
+                      <span className="italic">
+                        (by default the cheapest credit vintage will be
+                        purchased first)
+                      </span>
+                    </p>
+                    {creditVintages.map(({ date, credits }) => (
+                      <div
+                        key={date}
+                        className="h-auto flex flex-col items-start w-full p-15 border border-solid border-grey-300 mb-10 rounded-md"
+                      >
+                        <CheckboxLabel
+                          label={date}
+                          {...(form.register(`creditVintageOptions`),
+                          {
+                            value: date,
+                          })}
+                          onChange={handleCreditVintageOptions}
+                        />
+                        <p className="pl-[32px] m-0 mt-5">
+                          <span className="italic">
+                            {credits} credits available
+                          </span>
+                          <span className="px-5">|</span>
+                          <TextButton
+                            size="small"
+                            className="text-brand-300 uppercase bg-transparent border-none font-bold hover:opacity-80 text-sm"
+                          >
+                            view batch »
+                          </TextButton>
```

Comment:
include the link to the credit batch page, which is `/credit-batches/${batchDenom}`